apps/application-system/form/src/hooks/useAuth.ts

```diff
@@ -1,28 +1,30 @@
 import { setClientAuthToken } from '@island.is/application/graphql'
-import { useLocation } from 'react-router-dom'
+
 import { ActionType, useAuthState } from '../context/AuthProvider'
 import { userManager } from '../utils/userManager'
 
 const useAuth = () => {
   const [{ userInfo, userInfoState }, dispatch] = useAuthState()
-  const { pathname, search } = useLocation()
+
   async function signInUser() {
     dispatch({
       type: ActionType.SET_USER_PENDING,
     })
 
     try {
       const user = await userManager.verifyAuthentication()
+
       if (user) {
         setClientAuthToken(user.access_token)
+
         dispatch({
           type: ActionType.SET_USER_FULFILLED,
           payload: user,
         })
       }
     } catch {
       await userManager.signinRedirect({
-        state: { redirect: `${pathname}${search}` },
+        state: window.location.pathname,
```

Comment:
I'm not sure why the `search` was in there, but I guess it is fine to remove it?

Reply:
It was added in [#2291](https://github.com/island-is/island.is/pull/2291/)